<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CookiesConfigurationDialog</class>
 <widget class="QDialog" name="CookiesConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure cookies</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure cookies&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line17">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Accept Cookies:</string>
       </property>
       <property name="buddy">
        <cstring>acceptCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="acceptCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select the accept policy</string>
       </property>
       <item>
        <property name="text">
         <string>Always</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Never</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Only from sites you navigate to</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="exceptionsButton">
       <property name="toolTip">
        <string>Show a dialog to configure exceptions</string>
       </property>
       <property name="text">
        <string>&amp;Exceptions...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Keep until:</string>
       </property>
       <property name="buddy">
        <cstring>keepUntilCombo</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="keepUntilCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select the keep policy</string>
       </property>
       <item>
        <property name="text">
         <string>They expire</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>I exit the application</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>At most 90 days</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="cookiesButton">
       <property name="toolTip">
        <string>Show a dialog listing all cookies</string>
       </property>
       <property name="text">
        <string>&amp;Show Cookies...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="filterTrackingCookiesCheckbox">
       <property name="toolTip">
        <string>Select to filter tracking cookies</string>
       </property>
       <property name="text">
        <string>&amp;Filter Tracking Cookies</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>acceptCombo</tabstop>
  <tabstop>exceptionsButton</tabstop>
  <tabstop>keepUntilCombo</tabstop>
  <tabstop>cookiesButton</tabstop>
  <tabstop>filterTrackingCookiesCheckbox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CookiesConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CookiesConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
